src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -0,0 +1,44 @@
+using Neo.IO;
+using Neo.Json;
+using Neo.Persistence;
+using Neo.SmartContract.Native;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class Conflicts : TransactionAttribute
+    {
+        /// <summary>
+        /// Indicates the conflict transaction hash.
+        /// </summary>
+        public UInt256 Hash;
+
+        public override TransactionAttributeType Type => TransactionAttributeType.Conflicts;
+
+        public override bool AllowMultiple => true;
+
+        public override int Size => base.Size + Hash.Size;
+
+        protected override void DeserializeWithoutType(ref MemoryReader reader)
+        {
+            Hash = reader.ReadSerializable<UInt256>();
+        }
+
+        protected override void SerializeWithoutType(BinaryWriter writer)
+        {
+            writer.Write(Hash);
+        }
+
+        public override JObject ToJson()
+        {
+            JObject json = base.ToJson();
+            json["hash"] = Hash.ToString();
+            return json;
+        }
+
+        public override bool Verify(DataCache snapshot, Transaction tx)
+        {
+            return !NativeContract.Ledger.ContainsTransaction(snapshot, Hash);
```

Comment:
Isn't it better to change the attribute name to `RequireTransaction` and add a true or false to check if it is required to exist or not?

Reply:
For us is the same, just a boolean and is more flexible

Reply:
`Conflicts` is already used by NeoFS networks, there'll be severe compatibility issues if we're to change it. But we can have another attribute type for that purpose (hi, https://github.com/neo-project/neo/issues/1502#issuecomment-611550437) if we want to, we still have a lot of numbers for attributes. Also it'd be easier to handle then and less error-prone, making a mistake in boolean is more likely than making a mistake in a named type (and such mistakes could have quite some cost to them).